doc-content/jbpm-docs/src/main/asciidoc/CoreEngine/event-listeners-development-con.adoc

```diff
@@ -0,0 +1,17 @@
+[id='event-listeners-development-con_{context}']
+= Practices for development of event listeners
+
+The {PROCESS_ENGINE} calls event listeners during processing of events or tasks. The calls happen within {PROCESS_ENGINE} transactions are committed and block execution. Therefore, the event listener can affect the logic and performance of the {PROCESS_ENGINE}.
```

Comment:
Something missing here:
The calls happen within {PROCESS_ENGINE} transactions are committed and block execution. 
and are committed?

Reply:
The words "are committed" were wrong and have now been removed.